scripts/include_folder.py

```diff
@@ -28,9 +28,19 @@ def remove_ddeb_from_changes_file(filename):
             if not l.strip().endswith('.ddeb'):
                 trimmed_version.append(l)
             else:
+                deb_path = os.path.dirname(filename)
+                # Checksum entries have three columns. Files entries have four.
+                # The last is always the filename.
+                ddeb_file = l.strip().split(' ')[-1]
+                deb_file = ddeb_file.replace('.ddeb', '.deb')
```

Comment:
I have a slight concern with this blind renaming of `.ddeb` to `.deb` (and below).  Is it possible for the middle of a package name to have that string in it?  If so, we might be better off using a regex and only applying this change to the end.  What do you think about that?

Reply:
yeah, that occurred to me as well and I'm glad you asked because if I was more familiar with Python I would have figured out a more accurate solution.

In a regular scripting context I would use a regular expression as the matching pattern so that the substitution was limited. If python has an extname function we could also break off the extension from the basename and re-apply it to the .deb file extension.

Reply:
Yeah, python has `os.path.splitext` to split off just the extension; I think that would work well.